src/main/java/org/amahi/anywhere/adapter/FilesFilterAdapter.java

```diff
@@ -97,6 +97,14 @@ public void replaceWith(ServerShare serverShare, List<ServerFile> files) {
         notifyDataSetChanged();
     }
 
+    public ForegroundColorSpan getFcs() {
+        if (AmahiApplication.getInstance().isLightThemeEnabled()) {
+            return new ForegroundColorSpan(Color.parseColor("#328ce7"));
```

Comment:
Rather move the color codes to `colors.xml`

Reply:
I tried doing so, but that would require context to be present in the adapter. Since no context is present in `FilesFilterAdapter.java` I chose to hardcode color value.

Reply:
You can get the context in an adapter. Check that once. 
https://stackoverflow.com/questions/32136973/how-to-get-a-context-in-a-recycler-view-adapter

Reply:
I checked the link but 
1. `FilesFilterAdapter.java` is an abstract class so I cannot pass context in the constructor.
2. It also does not contain any view from which I can get context.

Reply:
I think I should move `getFcs()` into ServerFilesAdapter and ServerFilesMetadataAdapter then it will work fine

Reply:
Yeah try doing that.

Reply:
I have done it.